src/core/seat/input-manager.cpp

```diff
@@ -52,10 +52,52 @@ void wf::input_manager_t::handle_new_input(wlr_input_device *dev)
     data.device = nonstd::make_observer(input_devices.back().get());
     wf::get_core().emit(&data);
 
-    refresh_device_mappings();
+    configure_input_devices();
 }
 
-void wf::input_manager_t::refresh_device_mappings()
+void wf::input_manager_t::configure_input_device(std::unique_ptr<wf::input_device_impl_t> & device)
+{
+    auto dev     = device->get_wlr_handle();
+    auto cursor  = wf::get_core().get_wlr_cursor();
+    auto section =
+        wf::get_core().config_backend->get_input_device_section("input-device", dev);
+
+    auto mapped_output = section->get_option("output")->get_value_str();
+    if (mapped_output.empty())
+    {
+        if (dev->type == WLR_INPUT_DEVICE_POINTER)
+        {
+            mapped_output = nonull(wlr_pointer_from_input_device(
+                dev)->output_name);
+        } else if (dev->type == WLR_INPUT_DEVICE_TOUCH)
+        {
+            mapped_output =
+                nonull(wlr_touch_from_input_device(dev)->output_name);
+        } else
+        {
+            mapped_output = nonull(dev->name);
+        }
+    }
+
+    auto cal = section->get_option("calibration")->get_value_str();
+    if (!cal.empty())
+    {
+        device->calibrate_touch_device(cal);
+    }
+
+    auto wo = wf::get_core().output_layout->find_output(mapped_output);
+    if (wo)
+    {
+        LOGD("Mapping input ", dev->name, " to output ", wo->to_string(), ".");
```

Comment:
should this be LOGC(INPUT_DEVICE)? we can leave it like this if you think it is better

Reply:
I think leaving it this way makes it easier to see without the additional noise of `-d input-devices` and it is the same as it was before, so users can still use `-d` alone to see this particular output logged.